plugins/qeta-backend/src/database/DatabaseQetaStore.ts

```diff
@@ -544,6 +545,30 @@ export class DatabaseQetaStore implements QetaStore {
     });
   }
 
+  async postAttachment(
+    locationType: string,
+    binaryImage: Buffer,
+  ): Promise<Attachment[]> {
+    const attachments: Attachment[] = await this.db
+      .insert(
+        {
+          locationType,
+          binaryImage,
```

Comment:
This should only set binaryImage to database if the config `storage.type` is `database`. We don't want to save the file both in the filesystem and in database.